RPLCD/lcd.py

```diff
@@ -56,10 +62,22 @@ def __init__(self, cols=20, rows=4, dotsize=8, charmap='A02', auto_linebreaks=Tr
             auto_linebreaks:
                 Whether or not to automatically insert line breaks.
                 Default: True.
-
+            compat_mode:
+                Whether to run additional checks to support older LCDs
+                that may not run at the reference clock (or keep up with it).
+                Default: False
+            compat_mode_wait_time: Minimum time to pass between sends.
+                if zero, turns off compat_mode
+                Default: ``0.001`` seconds.
```

Comment:
Since `compat_mode=False` and `compat_mode_wait_time=0` are essentially the same, can we merge the two parameters?

Reply:
Yes, I'd like to. But there are two problems with having just the wait amount.
We'll lose backwards compatibility. That's ok. I can do an assert that tells would be users what to do instead.
There would be no default for compat_mode "on". The default amount would have to be sourced from something like DEFAULT_COMPAT_WAIT_TIME constant or the docs.
It's undoubtedly cleaner and gets rid of invalid configs like compat_mode=False with a high compat_mode_wait_time value. Your pick :wink: